### DISCUSSION TITLE
TeamTalk QT Client v5.14 for Windows: Unable to execute file, TeamTalk5.exe create process failed

### DISCUSSION THREAD

**Discussion by trancer1994** [BearWare/TeamTalk5#1995] [Q&A]
### Description

Hi,

When I try to install the new version of Teamtalk (v5.14 I believe, or at least that's what it has on the website) I select the components I want, which is normal. I then do the other stuff the installer asks. I then hit install once I'm done. This is when the problem starts. It asks me if I want to launch the newly-installed TeamTalk client (the box is ticked by default). I then hit finish, but it then throws an error: Unable to execute file dialog, teamtalk5.exe create process failed; code 2: the system cannot find the file specified. 

This happens both when I try to install with just the client and when I try to install with both the client and server. I've downloaded the installer properly and nothing seems to have been corrupted or anything like that. However, on the site it says underneath the link to download the installer that it's size is 46 MB, yet the downloaded file is only 44.3 MB, and it appears to be the same version. The only other thing I can do is try again but without launching TeamTalk with the tick box and then hitting finish and seeing what that does. I'm going to get a friend of mine to help me with it in the morning her time (she's in Canada). I've never had this happen before, ever. It's urgent I install it since I have servers I use every day which I want to go onto, private ones mainly. 

Please fix this bug, I'm not sure how many other people it is affecting. It could also be a problem with my computer but that's highly unlikely since Classic worked just fine.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

I expected the setup window to launch the newly-installed TeamTalk after I hit finish, then to close. 

### Actual behavior

Once I hit finish after completing all the installation steps, the setup window displays the error I talked about in the issue, then when I hit OK it just closes. 

### Steps to reproduce problem

You must be running Windows to reproduce this bug. 

Step 1. Go to the Download TeamTalk 5 section of the Bearware website if you don't have TeamTalk already or (very important) uninstall TeamTalk if it is already on the computer. 
Step 2. Then, click on the first file at the top of the table where it gives you the installers for all the different platforms (Windows, Mac OS, Linnux etc).
Step 3. Wait for the Windows installer to download. Click on it to open it, or just click Open or Run when prompted. 
Step 4. Follow the steps it presents you with on-screen. 
Step 5. When it asks you which components you want to install, choose either TeamTalk 5 Client or TeamTalk 5 Client And Server (it happens when I install both. Give both a try, but you will have to open the installer twice; repeat steps 3 through 5 to do this, but when you get to the components dialog, select the second option if you have already tried the first and vice versa). 
Step 6. When it asks you what other tasks you wish setup to perform, tick both boxes. 
Step 7. When it asks you which start menu folder you want to install it in, click Next. Important! Do not tick the box that says Don't Create A Start Menu Folder. 
Step 8. Now, after you've done all that, it should present you with the last step. Click Install. The application will now begin installing, depending on the choice you selected at the beginning. 
Step 9. When the application has finished installing, leave the box "Launch TeamTalk" ticked. Now hit finish. 
Step 10. The error I described in this issue should now present itself. Hit OK and wait for the installer to close. 
Step 11. Look on your desktop. If TeamTalk 5 is there, click on it to open it. A "missing shortcut, Windows is searching for TeamTalk5.exe" or similar should now come up. Click Cancel. If TeamTalk 5 is not there, repeat the steps 3 through 9, but in step 5 pick the second option.

**Comment by bear101** [comment]
Which version of Windows are you using? Can you find TeamTalk in Programs-menu?

**Comment by trancer1994** [comment]
I am using Windows 10 32-bit x86. When it has that error, everything has been installed properly, it's just that the program itself won't launch. If it's completely uninstalled, it uninstalls everything as it should, but when I try to install it it installs everything except the executable, it seems.

**Reply by bear101** [reply]
We only provide an executable for Windows 64-bit. I didn't even know a 32-bit Windows 10 existed.

**Comment by trancer1994** [comment]
And yes, TeamTalk 5 is in the start menu programs folder as well, under the Teamtalk 5 folder.

**Comment by CoBC** [comment]
Yes, 32 bit Windows has beene stopped with Windows 11. But in any ways, 
Qt isn't compatible with 32 bit systems so TT cannot work on it

**Comment by trancer1994** [comment]
Ah crap, so that's why it wasn't working. 

Then again, every version (even of the Classic) after 5.10 brought up that same error upon installation, whereas 5.10 and below worked just fine.

**Comment by trancer1994** [comment]
I was using the same OS for all of the versions I tried, 32-bit Windows 10.

**Comment by CoBC** [comment]
Yes, it's mensionned in change log of V5.11. We changed to Qt6 and Qt6 
only support 64 bit Windows, so since this change TT is only available 
for 64 bit OS
